### ISSUE TITLE
feature or issue?

### ISSUE THREAD

**Issue by egonw** [cdk/cdk#1064]
@johnmay, I really do not know if this is a bug or a feature, but I just notice this. Given the following two SMILES, depiction in CDK Depict doesn't work for the second:

```
[C@@H]1(OC([C@H](O)[C@H]([C@@H]1O)O)O*)CO
[C@@H]1(O[C@H]([C@H](O)[C@H]([C@@H]1O)O)O*)CO
```

![image](https://github.com/cdk/cdk/assets/26721/a2c44866-a624-4715-84ad-b76effbb63c3)

I am attempting to define the stereo of the last stereocenter, just as a visual check, so that I can attach it as R group to various T2 toxins, see https://social.edu.nl/@egonw/112302962828702865

**Comment by johnmay** [comment]
I presume it works if you don’t annotate the CIP labels.

CIP rules do not say anything about undefined atoms. The code tries to label as much as it can leave centres which may be R or S as unlabelled.

Note this does not mean they are undefined but a global label has no meaning. I’ll take a look as it should not throw and exception

**Comment by egonw** [comment]
> CIP rules do not say anything about undefined atoms. The code tries to label as much as it can leave centres which may be R or S as unlabelled.

Ah, I did not mean about the CIP (something we did indeed discuss before), but should it throw an exception?

**Comment by johnmay** [comment]
It's the CIP throwing the exception.

**Comment by johnmay** [comment]
I need to find time to look at it, probably a simple fix.

**Comment by egonw** [comment]
> I need to find time to look at it

not urgent

**Comment by johnmay** [comment]
Seems to be fixed now

**Comment by egonw** [comment]
Ah, I have the SMILES in https://chem-bla-ics.linkedchemistry.info/2025/01/05/sr24-results.html and just tried the latest CDK Depict and it indeed seems to work :)

Did you also release a new docker? cc @ozancinar

**Comment by ozancinar** [comment]
Is there a new container release for this? The last Docker container is from a year ago in the [list](https://hub.docker.com/r/simolecule/cdkdepict/tags) on the Dockerhub, which is the version 1.10. We are running the version 1.9.3; if you want we can make that update but I'm not sure if the fix from today will be in it either.

**Comment by johnmay** [comment]
We don't provide a docker container for the library? Do you mean cdk/depict?

**Comment by johnmay** [comment]
I'm still midway through doing the release cdk/depict will update at somepoint and I will do the container on docker hub as well.

**Comment by ozancinar** [comment]
Yes, I believe it is the container for the cdk/depict. I meant the container with the name cdkdepict under the user name simolecule to be precise.

**Comment by ozancinar** [comment]
Sorry, didn't mean to cause a confusion.

**Comment by egonw** [comment]
> please don't hijack existing issues as well.

dear both, my deepest apologies for this. I should have opened a separate ticket.

**Comment by egonw** [comment]
> Seems to be fixed now

Yes, agreed. And now closing the issue.

**Comment by johnmay** [comment]
@egonw sorry I didn’t see you tagged and was mid way through doing a release so was a bit short
